### PR TITLE
Adds interactive language buttons to getting started page

### PR DESCRIPTION
**Pull request by TisVictress**
I can confirm in the inspector that when a language button is clicked, the classes for each of the notes gets updated. But it's not showing on the webpage.
I'm using the same approach as the `View Output` functionality. Clicking a language button adds a class to the appropriate element, setting an override for the display to `block` and setting everything else to `none`.

cc @ryanmoran @shanks3012 @menehune23 to offer any suggestions :)

### CODE REVIEW

**Comment by TisVictress** [comment]
Also ccing @fg-j in case she has suggestions as well

File: assets/css/shame.scss [hunk 1]
```diff
@@ -67,4 +67,18 @@ code {
...
+once other the kpack and spring boot platforms are added */
+.getting-started__selections {
+	display: none;
+}
+
+.getting-started--selected {
+	.getting-started__selections {
+		display: block;
+	}
+}
```
Comment (ryanmoran) [inline assets/css/shame.scss:84]:
```suggestion
.getting-started__selections.getting-started--selected {
  display: block;
}
```

File: layouts/shortcodes/getting-started.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            if (notes[i].className.includes('getting-started--selected')) {
+                notes[i].classList.toggle('getting-started--selected');
+            }
+            // alert('after: ' + notes[i].className);
+        }
+        // alert(document.getElementById(id + '-notes').className);
+        document.getElementById(id + '-notes').classList.toggle('getting-started--selected');
+        // alert('after: ' + document.getElementById(id + '-notes').className);
+    }
+</script>
```
Comment (ryanmoran) [inline layouts/shortcodes/getting-started.html:43]: For this type of thing, its worth checking out the pattern we have in the `assets/js` directory. For example, here is the JS code that handles hiding/showing the docs sidebar: https://github.com/paketo-buildpacks/paketo-website/blob/de1232de6c46250586896fe2b8e5f1a7f89064aa/assets/js/components/docs-sidebar.js

File: assets/css/shame.scss (hunk 1 above)
Comment (ryanmoran) [reply assets/css/shame.scss:84]: This should unblock the hide/show issue, but I think you have a bug in the JS code.

Comment (ryanmoran) [reply assets/css/shame.scss:84]: Oh, let me explain the change too. CSS selectors cannot be nested like you have. You need to chain them. The selector I have changed this to basically says the element with both the `getting-started__selections` and `getting-started--selected` classes.

**Comment by TisVictress** [comment]
This PR resolves https://github.com/paketo-buildpacks/paketo-website/issues/270

**Pull request merged**